.github/workflows/bugs-triage-slack-notification.yml

```diff
@@ -3,14 +3,13 @@ name: Bugs Slack Notification
 on:
   issues:
     types:
-      - opened
+      - labeled
 jobs:
-  on-issue-opened:
+  on-issue-labeled:
     runs-on: ubuntu-22.04
     timeout-minutes: 5
     if: |
-        contains(github.event.issue.labels.*.name, '.Needs Triage') &&
-        contains(github.event.issue.labels.*.name, 'Type:Bug')
+      github.event.label.name == '.Needs Triage' && contains(github.event.issue.labels.*.name, 'Type:Bug')
```

Comment:
- won't this only work if `.Needs Triage` is added after `Type:Bug` ?
- won't this break the automation for newly opened issues? or does issue opening trigger labeled events too?

Reply:
@iethree  The issue here lies in how GitHub works. When we create a new issue from a template, at least three events are triggered: 1 - an event for opened and one event for each label (2 for the bug creation template). As a result, we get multiple triggers. To address this, I rely on a single event that is triggered in both cases: when an issue is opened and when labels are set.

However, yes, if Needs Triage is added first and then Type:Bug later, this won’t work. But it seems we’re okay with not supporting this case for now. If needed, we can handle it in a separate iteration.